AmethystAPI/src/amethyst/runtime/minecraft/MinecraftInfo.h

```diff
@@ -0,0 +1,19 @@
+#include "minecraft/src-deps/core/semVer/SemVersion.h"
+#include <string>
+#include <winrt/Windows.ApplicationModel.h>
+
+namespace Amethyst {
+    class MinecraftInfo {
```

Comment:
I'm not too fond on the MinecraftInfo name, as it is not very specific. Maybe something like MinecraftVersion would be better. Open to suggestions

Reply:
I talked to duck, we came up with `MinecraftPackageInfo`, might that be better?

Reply:
Yup that's great